src/Smidge/SmidgeStartup.cs

```diff
@@ -195,7 +195,7 @@ private static async void FileWatchOptions_FileModified(CacheBusterResolver cach
 
             //this file is part of this bundle, so the persisted processed/combined/compressed  will need to be 
             // invalidated/deleted/renamed
-            foreach (var compressionType in new[] { CompressionType.deflate, CompressionType.gzip, CompressionType.none })

```

Comment:
I think with the casing change on the enum this will effect some things. Here it will be for files. That's fine though, I'll have a look through the code to make sure this is lower cased where necessary.